src/components/icons/PerronIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ComponentWithAs } from '@chakra-ui/system';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+export const ChevronDownLargeBoldIcon: ComponentWithAs<'svg', IconProps> =
+  createIcon({
+    displayName: 'PerroIcon',
+    viewBox: '0 0 24 24',
+    path: (
+      <>
+        <title>Chevron Down Large Bold Icon</title>
+        <path
+          d="M5 9.41406L6.415 7.99906L12.0002 13.5842L17.5888 8.00299L18.9999 9.41412L12 16.4141L5 9.41406Z"
+          fill="currentColor"
+        />
```

Comment:
Not sure if you already did, but can we run this through [SVGo](https://svgo.dev/)? Just to make sure svg is optimized, thanks!

Reply:
Sorry, this icon doesn't exist, I just replace it with the one we already have